### PR TITLE
fix: remove unused CONTAINER_ORCHESTRATOR_ENABLED var

### PR DESCRIPTION
**Pull request by c-p-b**
## What
The behavior that this environment variable affects was removed in [this pr](https://github.com/airbytehq/airbyte/pull/13400/files). Therefore, this environment variable is no longer used and can be removed entirely.

## How
Remove all references to the environment variable from code and charts

### CODE REVIEW

File: airbyte-config/config-models/src/main/java/io/airbyte/config/EnvConfigs.java [hunk 1]
```diff
@@ -1043,11 +1042,6 @@ public Set<Integer> getTemporalWorkerPorts() {
     return Arrays.stream(ports.split(",")).map(Integer::valueOf).collect(Collectors.toSet());
   }
 
-  @Override
```
Comment (benmoriceau) [inline airbyte-config/config-models/src/main/java/io/airbyte/config/EnvConfigs.java:1046]: I think that is is missing an update to the `Configs.java` file as well

**Review by benmoriceau — APPROVED** [review]

File: airbyte-config/config-models/src/main/java/io/airbyte/config/EnvConfigs.java (hunk 1 above)
Comment (cpdeethree) [reply airbyte-config/config-models/src/main/java/io/airbyte/config/EnvConfigs.java:1046]: 👍

**Comment by rushil-somwanshi** [comment]
@cpdeethree Faced an error due to this PR where installation failed with the following error:

Error: INSTALLATION FAILED: template: airbyte/templates/env-configmap.yaml:67:42: executing "airbyte/templates/env-configmap.yaml" at <.Values.worker.containerOrchestrator.image>: nil pointer evaluating interface {}.image

Fixed it by adding back the following code in values.yaml file
`
worker.containerOrchestrator.image: ""
`

**Comment by git-phu** [comment]
this isn't right
container orchestrator is required for cloud and is used here: https://github.com/airbytehq/airbyte/blob/f76833e2e02c661141631fcab3e2eaabe5b19e88/airbyte-workers/src/main/java/io/airbyte/workers/config/ContainerOrchestratorConfigBeanFactory.java#L46-L47

**Pull request merged**